keyboards/planck/ez/config.h

```diff
@@ -139,3 +139,7 @@
 #define DRIVER_LED_TOTAL DRIVER_1_LED_TOTAL
 
 #define RGB_MATRIX_KEYPRESSES
+
+// If enabled, layer_state_set_kb will automatically enable
+// LEDs located under the space bar when current layer changes.
+#define PLANCK_EZ_CUSTOM_LEDS
```

Comment:
I would prefer that this isn't this isn't defined.  Because you'll need to undef it to remove it.  

In fact, could you add to the readme.md for the EZ instead, so that it notes that you can set this to disable the default behavior for the LEDs? 

```suggestion
```